src/components/ZoomBanner.component.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import './../styles/zoomwindow.css'
+import { debounce } from 'lodash'
```

Comment:
can you change this line to:
```jsx
import debounce from 'lodash/debounce'
```
lodash does imports/exports a little differently